install_config/install/host_preparation.adoc

```diff
@@ -293,10 +293,20 @@ accordingly.
 [[configuring-docker-storage]]
 == Configuring Docker Storage
 
-Containers and the images they are created from are stored in Docker's
-storage back end. This storage is ephemeral and separate from any
-xref:../../dev_guide/persistent_volumes.adoc#dev-guide-persistent-volumes[persistent storage] allocated to
-meet the needs of your applications.
+Containers and the images they are created from are stored in Docker's storage
+back end. This storage is ephemeral and separate from any
+xref:../../install_config/persistent_storage/index.adoc#install-config-persistent-storage-index[persistent
+storage] allocated to meet the needs of your applications. With _Ephemeral
+storage_, container-saved data will be lost once the container is removed. With
+_persistent storage_, container-saved data remains if the container is removed.
+
+Every system running a container daemon needs container storage configured. For
```

Comment:
maybe s/Every system running a container daemon needs container storage configured./You must configure storage for each system that runs a container daemon.